templates/4_Post_Clustering.ipynb

```diff
@@ -6,9 +6,10 @@
    "metadata": {},
```

Comment:
<p>Best to keep this consistent with the rest of this notebook, so let's use <code>'cell_table_thresholded.csv'</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>